releases.yml

```diff
@@ -1,3 +1,9 @@
+- version: 4.24.0
+  features:
+    - component: Linked logo section
+      url: /docs/patterns/linked-logo-section
+      status: New
+      notes: We've introduced a new pattern for linked logo sections
```

Comment:
Can you move this into 4.23.0, underneath pricing block? It's looking like they will release together